client/templates/regenerate.sh

```diff
@@ -0,0 +1,193 @@
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+set -e 
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+pushd "$SCRIPT_DIR/../python" > /dev/null
+
+#############################
+#      Delete old Tests     #
+#############################
+
+# List of test files to keep (from .../client/python)
+KEEP_TESTS=(
+  "test/test_cli_parsing.py"
+)
+
+find "test" -type f | while read -r file; do
+
+  # Check if file should be excluded
+  keep=false
+  for test_path in "${KEEP_TESTS[@]}"; do
+    if [[ "$file" == "$test_path"* ]]; then
+      keep=true
+    fi
+  done
+
+  if [ "$keep" = true ]; then
+    echo "${file}: skipped"
+    continue
+  else
+    echo "${file}: removed"
+    rm "${file}"
+  fi
+done
+
+#############################
+# Regenerate Python clients #
+#############################
+
+echo "Regenerating python from the spec"
+
+# TODO skip-validate-spec is needed because the upstream Iceberg spec seems invalid. e.g.:
+#   [main] ERROR o.o.codegen.DefaultCodegen - Required var sort-order-id not in properties
+
+docker run --rm \
+  -v "${SCRIPT_DIR}/../..:/local" \
+  openapitools/openapi-generator-cli generate \
+  -i /local/spec/polaris-management-service.yml \
+  -g python \
+  -o /local/client/python \
+  --additional-properties=packageName=polaris.management \
+  --additional-properties=apiNamePrefix=polaris > /dev/null 
+
+docker run --rm \
+  -v "${SCRIPT_DIR}/../..:/local" \
+  openapitools/openapi-generator-cli generate \
+  -i /local/spec/polaris-catalog-service.yaml \
+  -g python \
+  -o /local/client/python \
+  --additional-properties=packageName=polaris.catalog \
+  --additional-properties=apiNameSuffix="" \
+  --skip-validate-spec > /dev/null
+
+docker run --rm \
+  -v "${SCRIPT_DIR}/../..:/local" \
+  openapitools/openapi-generator-cli generate \
+  -i /local/spec/iceberg-rest-catalog-open-api.yaml \
+  -g python \
+  -o /local/client/python \
+  --additional-properties=packageName=polaris.catalog \
+  --additional-properties=apiNameSuffix="" \
+  --additional-properties=apiNamePrefix=Iceberg \
+  --skip-validate-spec > /dev/null
+
+#############################
+#      Prepend Licenses     #
+#############################
+
+echo "Re-applying license headers"
+
+prepend_header() {
+  local file="$1"
+  local header_file="$2"
+  tmpfile=$(mktemp)
+  cat "$header_file" "$file" > "$tmpfile"
+  mv "$tmpfile" "$file"
+}
+
+# List of paths to exclude (from .../client/python)
+EXCLUDE_PATHS=(
+  "./.gitignore"
+  "./.openapi-generator/"
+  "./.openapi-generator-ignore"
+  "./.pytest_cache"
+  "./test/test_cli_parsing.py"
+  "./cli/"
+  "./polaris/__pycache__"
+  "./polaris/catalog/__pycache__/"
+  "./polaris/catalog/models/__pycache__/"
+  "./polaris/catalog/api/__pycache__/"
+  "./polaris/management/__pycache__/"
+  "./polaris/management/models/__pycache__/"
+  "./polaris/management/api/__pycache__/"
+)
```

Comment:
Shall we add `./.DS_STORE` to exclude_paths for mac users? This makes running `regenerate.sh` fails in my mac locally with
```
Re-applying license headers
No header compatible with file ./.DS_Store
```

Reply:
```suggestion
   "./.DS_Store"
  "./poetry.lock"
)
```
Also "poetry.lock" should be excluded